libs/documentation/src/lib/components/sidenavigation/demos/links-side-navigation/links-side-navigation.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+import { INavigationLink, SdsDialogConfig, SdsDialogRef, SideNavigationModel } from '@gsa-sam/components';
+import { navigationConfig } from '../filter-side-navigation/navigation.config';
+
+@Component({
+  selector: 'links-side-navigation',
+  templateUrl: './links-side-navigation.component.html'
+})
+export class LinksSideNavigationComponent implements OnInit {
+
+  responsiveDialogOptions: SdsDialogConfig = {
+    ariaLabel: 'Navigation Links',
+  };
+
+  isMobileMode: boolean;
+  mobileDialog: SdsDialogRef<any>;
+
+  public navigationModel: SideNavigationModel = navigationConfig;
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  onDialogOpen($event) {
+    this.mobileDialog = $event;
+  }
+  onCancelClicked() {
+    this.mobileDialog.close();
+    this.mobileDialog = undefined;
+  }
```

Comment:
The two methods above don't seem to have any use as it's never called from template - `onDialogOpen` and `onCancelClicked`. Similarly, the fields isMobileMode and mobileDialog are never changed from undefined